### PR TITLE
Added Server handling checks

### PR DESCRIPTION
**Pull request by d3agle**
-Added some server handling checks to make sure handling of a client disconnection doesn't affect the stability of the server

### CODE REVIEW

File: Server/Core/Commands/SurveillanceHandler.cs [hunk 1]
```diff
@@ -68,6 +68,7 @@ public static void HandleGetProcessesResponse(Client client, GetProcessesRespons
                         ListViewItem lvi =
                             new ListViewItem(new string[] { packet.Processes[i], packet.IDs[i].ToString(), packet.Titles[i] });
 
+                        if (client.Value != null && client.Value.FrmTm != null)
                         client.Value.FrmTm.AddProcessToListview(lvi);
```
Comment (MaxXor) [inline Server/Core/Commands/SurveillanceHandler.cs]: Please put a tab-space here.

File: Server/Core/Commands/SurveillanceHandler.cs [hunk 2]
```diff
@@ -76,12 +77,13 @@ public static void HandleGetProcessesResponse(Client client, GetProcessesRespons
...
         public static void HandleGetKeyloggerLogsResponse(Client client, GetKeyloggerLogsResponse packet)
         {
-            if (client.Value.FrmKl == null)
+            if (client.Value == null || client.Value.FrmKl == null)
                 return;
 
             if (packet.FileCount == 0)
             {
-                client.Value.FrmKl.SetGetLogsEnabled(true);
+                if (client.Value != null && client.Value.FrmKl != null)
```
Comment (MaxXor) [inline Server/Core/Commands/SurveillanceHandler.cs]: This is redundant.

File: Server/Core/Commands/SystemHandler.cs [hunk 3]
```diff
@@ -29,12 +29,13 @@ public static void HandleGetDrivesResponse(Client client, GetDrivesResponse pack
                 drives[i] = new RemoteDrive(packet.DriveDisplayName[i], packet.RootDirectory[i]);
```
Comment (MaxXor) [inline Server/Core/Commands/SystemHandler.cs]: Line 21 is a missing `null`-check.

**Comment by d3agle** [comment]
Fixed thank you!

File: Server/Core/Commands/SurveillanceHandler.cs (hunk 2 above)
Comment (d3agle) [inline Server/Core/Commands/SurveillanceHandler.cs]: I know we are handling these exceptions in the methods, but for the static methods that are threaded shouldn't we need to check if Value is nulled before calling one of these methods?

**Comment by MaxXor** [comment]
Thanks, 800 commits :tada:

**Comment by d3agle** [comment]
:D and many more to come!

**Pull request merged**